docs/user-guide/estimators.rst

```diff
@@ -91,7 +91,7 @@ arguments are translated into a TS difference assuming ``ts = n_sigma ** 2``.
 .. _sedtypes:
 
 SED types
-~~~~~~~~~
+^^^^^^^^^
```

Comment:
The rest of section titles use `------`

Reply:
This is a subsection, as it belongs to the `Flux quantities` heading

Reply:
Ah, right, I did not realized that

Reply:
No worries!